sdk/eventhub/event-hubs/src/util/constants.ts

```diff
@@ -3,5 +3,5 @@
 
 export const packageJsonInfo = {
   name: "@azure/event-hubs",
-  version: "3.0.0-preview.1"
+  version: "3.0.0-preview.1.0"
```

Comment:
I think we're actually updating the version numbers to `5.0.0-preview.1.0`.

@ramya-rao-a Should the new version apply to `core-amqp` as well? Or is this only for the data-plane packages?

Reply:
All packages will be using `-preview.1`